### PR TITLE
chore(portals-my-pages): Move libs

### PR DESCRIPTION
**Pull request by thordurhhh**
## What

Move `libs/service-portal` to `libs/portals/my-pages` ✨ 

🏚️   ➡️   🏡 

## Why

Service portals and portals admin and all portals belong in the portals structure.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

## Release Notes

- **New Features**
  - Reorganized project structure to enhance navigation and ownership assignments, consolidating service portal components under the new "my-pages" namespace.

- **Improvements**
  - Updated import paths across various components to reflect the new module structure, ensuring consistent functionality and access to core features.

- **Documentation**
  - Updated README files and configuration settings to align with the new project naming conventions and directory structure.

- **Tests**
  - Adjusted test configurations and import paths to accommodate the updated project structure, maintaining test integrity and functionality.

### CODE REVIEW

**Review by Toti91 — APPROVED** [review]
Code owned LGTM

**Review by disaerna — APPROVED** [review]
Files look good and I have tested "my pages" on the feature branch ✅

File: libs/portals/my-pages/assets/jest.config.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default {
     ],
   },
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'html'],
-  coverageDirectory: '<rootDir>/coverage/libs/service-portal/constants',
+  coverageDirectory: '<rootDir>/coverage/libs/portals/my-pages/assets',
```
Comment (disaerna) [inline libs/portals/my-pages/assets/jest.config.ts:13]: Should be **constants** not **assets**

File: libs/portals/my-pages/assets/jest.config.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export default {
...
   globals: {},
-  displayName: 'service-portal-constants',
+  displayName: 'portals-my-pages-assets',
```
Comment (disaerna) [inline libs/portals/my-pages/assets/jest.config.ts:15]: Should be constants not assets

**Review by snaerseljan — APPROVED** [review]
Awesome job. Thank you so much for taking this on 🙏

File: apps/system-e2e/src/tests/islandis/service-portal/smoke/information.spec.ts [hunk 3]
```diff
@@ -2,8 +2,8 @@ import { BrowserContext, expect, test } from '@playwright/test'
 import { icelandicAndNoPopupUrl, urls } from '../../../../support/urls'
 import { session } from '../../../../support/session'
 import { label } from '../../../../support/i18n'
-import { m } from '@island.is/service-portal/core/messages'
-import { spmm } from '@island.is/service-portal/information/messages'
+import { m } from '@island.is/portals/my-pages/core/messages'
+import { spmm } from '@island.is/portals/my-pages/information/messages'
```
[the remark this answers is not in the document]
Comment (thordurhhh) [reply apps/system-e2e/src/tests/islandis/service-portal/smoke/information.spec.ts:6]: None of these files use the old import paths.
storageState is not import.

File: apps/portals/my-pages/tsconfig.app.json [hunk 4]
```diff
@@ -1,8 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "sourceMap": true,
-    "outDir": "../../dist/out-tsc",
+    "outDir": "../../../dist/out-tsc",
```
[the remark this answers is not in the document]
Comment (thordurhhh) [reply apps/portals/my-pages/tsconfig.app.json:4]: this is just wrong.

**Review by svanaeinars — APPROVED** [review]
LGTM

**Pull request merged**